modules/nw-sriov-dual-nic-con.adoc

```diff
@@ -4,11 +4,16 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="nw-sriov-dual-nic-con_{context}"]
-= NIC partitioning for SR-IOV devices
+= NIC partitioning for SR-IOV devices (Technology Preview)
 
 {product-title} can be deployed on a server with a dual port network interface card (NIC).
 You can partition a single, high-speed dual port NIC into multiple virtual functions (VFs) and enable SR-IOV.
 
+[NOTE]
```

Comment:
Do we keep or remove this note now @cgoncalves ? It was previously removed but I added back in when I reverted the TP status changes.

Reply:
It is possible with CNF-6356 but Tech Preview, too.

Reply:
Then I will remove this note. Since we spoke, CNF-6356 is back to GA so I think it's ok to remove.